### ISSUE TITLE
Are there any pre-trained models for AnomalyDetector

### ISSUE THREAD

**Issue by volodymyrkepsha** [ekosman/AnomalyDetectionCVPR2018-Pytorch#43]
Hi, awesome work !
I saw that there are weight for C3D net, but are the any ready-to-use models for AnomalyDetector ?
If you have at least some pre-trained weights for AnomalyDetector, please share it. 
Thanks

**Comment by ekosman** [comment]
I have some trained models in "exps." I also trained the model for longer periods and will upload them soon along with the loss graph

**Comment by volodymyrkepsha** [comment]
@ekosman 
Thanks for the quick respond !
Sorry, didn't notice that there were models. That's exactly what I need. 
Thanks for your work !
